src/cache.jl

```diff
@@ -113,7 +113,7 @@ function updatecache(absentmodules::Vector{Symbol}, server)
     
     @async begin 
         str = readline(o)
-        data = base64decode(str)
+        data = base64decode(chomp(str))
```

Comment:
`base64decode` doesn't work with a SubString, 
`data = base64decode(String(chomp(str)))`
gets this working. We reallocated the full string